Glyssen/Dialogs/ExportDlg.cs

```diff
@@ -159,6 +159,10 @@ private void BtnOk_Click(object sender, EventArgs e)
 			try
 			{
 				m_projectExporter.GenerateFile(filePath, m_selectedFileType);
+
+				// remember the location
+				Settings.Default.LastExportLocation = Path.GetDirectoryName(filePath);
```

Comment:
I think this needs to be per project, thus needs to be saved in the project metadata.